core/src/main/java/org/jboss/jca/core/connectionmanager/tx/TxConnectionManagerImpl.java

```diff
@@ -386,7 +388,15 @@ public long getTimeLeftBeforeTransactionTimeout(boolean errorRollback) throws Ro
          return ((TransactionTimeoutConfiguration)transactionManager).
             getTimeLeftBeforeTransactionTimeout(errorRollback);  
       }
-      
+
+      if (transactionManager instanceof ContextTransactionManager)
```

Comment:
@maeste the readme states that this branch is intended exclusively for wildfly, so I think ContextTransactionManager should always be present on classpath (i.e. this is driven by jboss-moduless in wildfly, which I will handle). However, when porting this to master branch, should i check that ContextTransactionManager exists?

Reply:
OK, this looks like not backportable to master.

Reply:
@TomasHofman we will probably have to review this Jira as part of our work at the master branch. This branch contains the future version of IJ, 2.x, that will replace 1.x in WF in the future. The team is still working on it.